src/components/scenes/CreateWalletAccountSelectScene.tsx

```diff
@@ -15,7 +16,6 @@ import s from '../../locales/strings'
 import { PrimaryButton } from '../../modules/UI/components/Buttons/PrimaryButton.ui'
 import { FormattedText as Text } from '../../modules/UI/components/FormattedText/FormattedText.ui'
 import { Gradient } from '../../modules/UI/components/Gradient/Gradient.ui'
-import { SafeAreaViewComponent as SafeAreaView } from '../../modules/UI/components/SafeAreaView/SafeAreaView.ui'
```

Comment:
With this gone, you can fully delete `src/modules/UI/components/SafeAreaView/SafeAreaView.ui.tsx`.